activetable/cells.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+"""Classes representing table cells.
+
+These classes are used mainly as namespaces to dump all data associated with a cell.  There is no
+expectation that all attributes are set in __init__() or that attributes are controlled byt the
+classes themselves.
+"""
+from __future__ import absolute_import, division, unicode_literals
+
+import decimal
+
+
+class Cell(object):
+    """Abstract base class for all cells."""
+
+    def __eq__(self, other):
+        """Test for equality based on type and attribute values."""
+        return type(self) is type(other) and vars(self) == vars(other)
+
+
+class StaticCell(Cell):
+    """A static cell with a fixed value in the table body."""
+
+    is_static = True
+
+    def __init__(self, value):
+        self.value = value
+
+
+class NumericCell(Cell):
+    """A numeric response cell."""
+
+    is_static = False
```

Comment:
It'd be nicer to move this to Cell, and let StaticCell be the only one that needs to override it.